code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -110,6 +186,15 @@
     assembled = 2
     active = 0
 
+/obj/item/weapon/grenade/iedcasing/preassembled/withshrapnel
+	name = "shrapnel loaded improvised explosive"
+
+/obj/item/weapon/grenade/iedcasing/preassembled/withshrapnel/New()
+	..()
+	for(var/i = 1, i<=4,i++)
+		AddShrapnel(new /obj/item/weapon/shard(src), null)
+
```

Comment:
before i get called out on it:
ill make user=null the default so that i only have to
`AddShrapnel(new /obj/item/weapon/shard(src))`
here